src/str/combine_expansion_hunter_json_to_tsv.py

```diff
@@ -122,7 +122,14 @@ def main():
             try:
                 json_contents = parse_json_file(json_path)
             except Exception as e:
-                logging.info(f"Skipping {json_path}... Unable to parse json: {e}")
+                logging.warning(f"Skipping {json_path}... Unable to parse json: {e}")
+                continue
```

Comment:
Do you need the 2 `continue`'s here?

Reply:
I guess the first one `continue`s if the json is malformed (e.g., syntax or structural errors) and the second one `continue`s if the syntax is correct but it is missing a required field.